### DISCUSSION TITLE
Problem with styling, k bar looks very small and not correct

### DISCUSSION THREAD

**Discussion by amulyaparmar** [timc1/kbar#283] [Q&A]
<img width="1593" alt="image" src="https://user-images.githubusercontent.com/35022914/215339099-32af8634-57ae-4f87-8afe-601b1c94bf78.png">
Hi when i try to install kbar into a react chrome extension tmeplate:
https://github.com/lxieyang/chrome-extension-boilerplate-react

it does not work for me for some reason!

**Comment by emiliosheinz** [comment]
Looks like kbar comes without styles by default. For more information check this issue: https://github.com/timc1/kbar/issues/334
